repo/packages/M/minio/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "minio",
+  "version": "0.0.1-RELEASE.2016-09-11T17-42-18Z",
```

Comment:
version should be minioVersion-packagingVersion
Ex: If Minio  version is 1.2.3 then the `version` here will be `1.2.3-0.0.1`